app/src/main/java/com/duckduckgo/app/trackerdetection/blocklist/BlockListInterceptorApiPlugin.kt

```diff
@@ -51,10 +55,11 @@ class BlockListInterceptorApiPlugin @Inject constructor(
         }
 
         return activeExperiment?.let {
+            val config = activeExperiment.getSettings()?.let { jsonAdapter.fromJson(it) } ?: emptyMap()
             val path = when {
-                activeExperiment.isEnabled(TREATMENT) -> activeExperiment.getConfig()[TREATMENT_URL]
-                activeExperiment.isEnabled(CONTROL) -> activeExperiment.getConfig()[CONTROL_URL]
-                else -> activeExperiment.getConfig()[NEXT_URL]
+                activeExperiment.isEnabled(TREATMENT) -> config["treatmentUrl"]
+                activeExperiment.isEnabled(CONTROL) -> config["controlUrl"]
+                else -> config["nextUrl"]
             } ?: chain.proceed(request.build())
```

Comment:
Can you do me a favour and add a return here? :)

`} ?: return chain.proceed(request.build())`

Reply:
@aitorvs FYI I've just added this change myself because why not.